Update production doc (#532)
nest-desktop/nest-desktop: docs/developer/stages/production.rst

|br|

.. _production_python:

Python
------

> .. image:: /_static/img/logo/python-logo.svg

Reviewer: I receive the error
```
nest-desktop/docs/developer/stages/production.rst:: WARNING: image file not readable: _static/img/logo/python-logo.svg
```

Reviewer: Fixed with ee8932e

Conda
-----

> .. image:: /_static/img/logo/conda-logo.svg

Reviewer: I receive the error
```
nest-desktop/docs/developer/stages/production.rst:: WARNING: image file not readable: _static/img/logo/conda-logo.svg
```

Reviewer: Fixed with ee8932e

|br|

.. _production_electron:

> Electron (deb)

Reviewer: ```suggestion
Electron (`.deb` package)
```

> Electron (`.deb` package)
> -------------------------

Reviewer: ```suggestion
Electron (``.deb`` package)
---------------------------
```

In ``package.json``, there are also yarn commands configured to build an Electron app.

.. code-block:: bash

   yarn electron:build

> Then install the ``deb`` file on your linux system.

Reviewer: ```suggestion
Then install the ``.deb`` file on your Linux system.
```

.. seeAlso::

>    If you want to build another Electron packages, please have a look into ``electron-builder.yml`` file.

Reviewer: ```suggestion
   If you want to build other Electron packages, please have a look into ``electron-builder.yml`` file.
```

>    For snap you can find more information on `snap repository for nest-desktop <https://github.com/nest-desktop/nest-desktop-snap>`__.

Reviewer: ```suggestion
   For Snap packages, you can find more information in the `Snap repository for nest-desktop <https://github.com/nest-desktop/nest-desktop-snap>`__.
```
